open_by_laws/views/by_laws.py

```diff
@@ -1,5 +1,16 @@
+from django.shortcuts import get_object_or_404
+
 from liiweb.views.legislation import LocalityLegislationListView
+from peachjam.models import Locality
 
 
 class MunicipalByLawsView(LocalityLegislationListView):
     template_name = "open_by_laws/municipal_by_laws_list.html"
+    navbar_link = "legislation/municipal"
+
+    def get(self, *args, **kwargs):
+        self.municipality = get_object_or_404(Locality, code=kwargs["code"])
```

Comment:
Just a reminder that this is exactly what LocalityLegislationListView does, using the word `locality` rather than `municipality`. So this isn't necessary.